genesis/genesis.go

```diff
@@ -266,270 +264,17 @@ func FromFlag(networkID uint32, genesisContent string) ([]byte, ids.ID, error) {
 func FromConfig(config *Config) ([]byte, ids.ID, error) {
 	hrp := constants.GetHRP(config.NetworkID)
 
-	amount := uint64(0)
```

Comment:
This change is much huger than inplace editing, I suggest either:
- 1 liner call into camino file if depositBondMode is true
- 2 liner (camino bock and platformAllocations)
- Add our stuff into here (inplace)